server/src/main/java/edu/jhu/server/Game.java

```diff
@@ -60,9 +87,26 @@ public void handleEvent(JSONObject event) {
 	  		System.out.println("test event");
 	  		break;
 	  	case CHAT_NOTIFICATION:
-	  		System.out.println(event.getString("body"));
 	  		notifyPlayers(event);
 	  		break;
+	  	case SUGGESTION_REQUEST:
+	  		Player suggester = getPlayerByTag(event.getString("author"));
+	  		if (board.isSuggestionValid(suggester)) {
+	  			JSONObject chat = makeChatMessage(suggester.getTag() +
```

Comment:
A suggestion actually has a couple different notifications to it (e.g. the move notification) so instead of putting them in one giant notification, I broke them up into constituent parts.